vcbuild.bat

```diff
@@ -689,6 +689,9 @@ goto exit
 
 :create-msvs-files-failed
 echo Failed to create vc project files.
+if %VCBUILD_PYTHON_VERSION%==3 (
+  echo Python 3 is not yet fully supported, to avoid issues Python 2 should be installed.
```

Comment:
Nit that can totally be ignored, especially since this mess eventually will go away anyway.

```suggestion
  echo Python 3 is not yet fully supported. To avoid issues, Python 2 should be installed.
```

Reply:
Yikes, meant "this message wil go away" not "this mess"... fixed!  :-D